### PR TITLE
Fibonnaci Number in Kotlin

### PR DESCRIPTION
**Pull request by hemant2705**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
yes

### Description
The Fibonacci series is a series where the next term is the sum of pervious two terms.
The first two terms of the Fibonacci sequence is 0 followed by 1
So a program in kotlin to solve the same.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#5316 
(Write your answer here.)

### CODE REVIEW

File: Kotlin/Maths/FibonacciNumber.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/*
+The Fibonacci series is a series where the next term is the sum of pervious two terms.
+The first two terms of the Fibonacci sequence is 0 followed by 1.
+
+The Fibonacci sequence: 0, 1, 1, 2, 3, 5, 8, 13, 21, ...
+*/
+import java.util.Scanner                    //importing scanner
```
Comment (ankitaggarwal23) [inline Kotlin/Maths/FibonacciNumber.kt]: Remove the redundant comments

File: Kotlin/Maths/FibonacciNumber.kt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+fun main(args: Array<String>) {
+   
+   val reader = Scanner(System.`in`)       
+   print("Enter any number: ")  
+   var n :Int = reader.nextInt()            //Obtaining input from user
```
Comment (ankitaggarwal23) [inline Kotlin/Maths/FibonacciNumber.kt]: Remove the redundant comments

File: Kotlin/Maths/FibonacciNumber.kt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    var t1 = 0
+    var t2 = 1
+
+    print("Fibonacci series upto $n : ")
+
+    while (i <= n) {
+        print("$t1 + ")
+
+        val sum = t1 + t2
+        t1 = t2                      //swapping the terms
```
Comment (ankitaggarwal23) [inline Kotlin/Maths/FibonacciNumber.kt]: Remove the redundant comments

File: Kotlin/Maths/FibonacciNumber.kt [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+fun main(args: Array<String>) {
+   
+   val reader = Scanner(System.`in`)       
+   print("Enter any number: ")  
+   var n :Int = reader.nextInt()            //Obtaining input from user
+  
+    var i = 1
+    var t1 = 0
+    var t2 = 1
```
Comment (ankitaggarwal23) [inline Kotlin/Maths/FibonacciNumber.kt]: Don't use single letter for variable names use proper naming conventions

File: Kotlin/Maths/FibonacciNumber.kt (hunk 2 above)
Comment (ankitaggarwal23) [inline Kotlin/Maths/FibonacciNumber.kt]: Don't use single letter for variable names use proper naming conventions

File: Kotlin/Readme.md [hunk 5]
```diff
@@ -24,6 +24,8 @@
 - [Armstrong Number](Maths/Armstrong.kt)
 - [Armstrong Number with Recursive Approach](Maths/ArmstrongRecursive.kt)
 - [Palindrome Number](Maths/PalindromeNumber.kt)
+- [FibonacciNumber](Maths/FibonacciNumber.kt)
```
Comment (ankitaggarwal23) [inline Kotlin/Readme.md]: Rewrite it as `[Fibonacci Number]`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by hemant2705** [comment]
I have resloved all the changes suggested.
Please review it again sir.

File: Kotlin/Maths/FibonacciNumber.kt [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        term2 = sum
+
+        index++
+    }
+}
+
+/*
+Sample Test case:
+Enter any number: 5
+Fibonacci series upto 5: 0 + 1 + 1 + 2 + 3
```
Comment (ankitaggarwal23) [inline Kotlin/Maths/FibonacciNumber.kt]: why you are adding `+` either remove it or place `,` or just add a space

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by hemant2705** [comment]
I have replaced the+ sign with " , " as asked
please review it again.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**